apps/admin-portal/src/api/identity-provider.ts

```diff
@@ -25,7 +25,7 @@ import {
     IdentityProviderListResponseInterface,
     IdentityProviderResponseInterface,
     IdentityProviderTemplateListInterface,
-    IdentityProviderTemplateListItemInterface,
+    IdentityProviderTemplateListItemInterface, OutboundProvisioningConnectorInterface,
```

Comment:
```suggestion
    IdentityProviderTemplateListItemInterface,
    OutboundProvisioningConnectorInterface,
```

Reply:
This is addressed in a separate PR: https://github.com/wso2/identity-apps/pull/692/commits/fd488a117de604742a197f69196184e1e1c4e951